app/controllers/api/vms_controller.rb

```diff
@@ -152,6 +159,7 @@ def scan_resource(type, id = nil, _data = nil)
         result
       end
     end
+    central_admin :scan_resource, :scan
```

Comment:
There are a few of these in here that I don't think we support from the main app.

Doing a quick look for `api_relay` in the core source I can only find `shutdown_guest`, `reboot_guest`, `reset`, `start`, `stop`, and `suspend`. We should keep the actions we support consistent between the UI and API.

Reply:
Yeah, they were simple, so I added them.

Reply:
Also, I'd like to add all future central admin features through the api and maybe even deprecate the central admin support in the models.

Reply:
I wouldn't introduce new behavior just to make it complete.  We can add the new ones later when they're needed.

Reply:
I still think we should leave out the extra methods here until they are also supported through the classic UI. If this PR is meant to solve a particular BZ we should limit the scope that issue rather than also adding new features.

Reply:
Since enabling them all is so easy, I figured I would just do it proactively rather than waiting for another high severity customer case saying Action X doesn't work.

Reply:
Right, but as I understand it this was because of missing operations in the API vs the classic UI. If you really do want to add the new ones, then please create an additional PR to add the same operations in core for consistency.

Reply:
I can see the value in separating the bugfix from the enhancement part of this PR. 👍

Reply:
Okay, I removed the things we don't central admin in the UI.  I'll open a followup PR to add those.